### PR TITLE
Adding initial setup guide

### PR DESCRIPTION
**Pull request by rr1000**
A comprehensive initial setup guide. This guide is intended to be a one stop shop for getting setup on Stratum.

### CODE REVIEW

File: initial-setup.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+Once inside you can run the `catalyze associate` command. This command takes two arguments: `ENV_NAME` and `SERVICE_NAME`. The `ENV_NAME` is the name of the provisioned environment that you want associated with the application you're located inside of. The `ENV_NAME` is located on your contract as well as inside the Stratum dashboard. The `SERVICE_NAME` is the name of the related code service. We automatically name this for you so in most cases you'll be looking for `app01`.
+
+**Full example:** `catalyze associate DemoProd app01`
+
+**Recap:** Run the `catalyze associate ENV_NAME SERVICE_NAME [-a] [-r] [-d]` command to associate your application repository with your Catalyze remote repository.
+
+## SSH Keys
+
+The next thing you'll want todo is setup your SSH key so you can push code to your Catalyze remote repository. Any SSH key uploaded to your Catalyze user account will be available for use with all code services and environments that you have access to.
```
Comment (josomo) [inline initial-setup.md]: to_do

File: initial-setup.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+---
+title: Stratum Initial Setup
+---
+
+## Introduction
+
+The following article is intended to get new users up and running in short order. We do have verbose, in depth write ups on each step outlined below. If you'd like to read more please see our official [getting started guide](//resources.catalyze.io/stratum/getting-started/). However it is recommended that you complete this guide before moving on to anything else.
+
+What we'll complete in this guide:
```
Comment (josomo) [inline initial-setup.md]: complete = cover?

File: initial-setup.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+**Please note:** If for any reason you find yourself stuck you can run any Catalyze CLI command without arguments to see a full manual on that specific command.
+
+**Recap:** Use a variation of the `catalyze certs create` command to upload your SSL certificate.
+
+## Push Code
+
+The moment of truth. It's time to make our first code push, and it couldn't be easier. Simply navigate to application's code repository (the same place we were in the associate step) and run `git push catalyze master`.
+
+This, of course, your master branch to the Catalyze master branch. If you want to push a branch that is not named master, see the following:
```
Comment (josomo) [inline initial-setup.md]: pushes your master branch

**Comment by josomo** [comment]
lgtm

**Comment by rr1000** [comment]
Make a note about adding the newly created ssh key to ssh agent `ssh-add /Users/ryanSrich/catalyze_rsa`

**Pull request merged**